apps/web/src/pages/index.tsx

```diff
@@ -90,4 +90,29 @@ function Home() {
   );
 }
 
+export async function getServerSideProps() {
+  let isBridgeUp = true;
+
+  return fetch(`https://wallet.defichain.com/api/v0/bridge/status`)
+    .then((res) => Promise.all([res.json(), res.status]))
+    .then(([data, statusCode]) => {
+      if (statusCode === 200) {
+        if (data?.isUp === false) {
+          isBridgeUp = false;
```

Comment:
```suggestion
          isBridgeUp = data?.isUp === false;
```

No need for the if statement anymore.